### PR TITLE
Docker Hub changes

### LINKED ISSUE #4: Docker hub R-base -> python
When I ran this with my test group, they thought we should show the python docker hub page instead of R since we use python for the rest of the lesson.  I did when I taught it but I think it might be helpful to add more examples of dockerhub pages to the lessons to give them more than one example so maybe we could show both?

### PR DESCRIPTION
**Pull request by ChristinaLK**
Closes #4 and #16.

### CODE REVIEW

**Comment by ChristinaLK** [comment]
@sstevens2

File: _episodes/04-docker-hub.md [hunk 1]
```diff
@@ -36,6 +47,91 @@ The main body of the page contains many used headings, such as:
...
+```
+$ docker pull python:3.6
+```
+{: .bash}
+
+The default tag (which is used if you don't specify one) is called `latest`.
+
+So far, we've only seen containers that are maintained by the Docker team. However,
+it's equally common to use containers that have been produced by individual owners
+or organizations. The most useful R containers are developed by an organization called
```
Comment (sstevens2) [inline _episodes/04-docker-hub.md:79]: The R comment seems a bit disconnected, maybe add into a callout instead?

Comment (sstevens2) [reply _episodes/04-docker-hub.md:79]: or maybe putting this in the paragraph with the challenge?

File: _episodes/04-docker-hub.md [hunk 2]
```diff
@@ -36,6 +47,91 @@ The main body of the page contains many used headings, such as:
...
+## Many Different Containers
+
+There are many different containers on Docker Hub. This is where the real advantage
+of using containers shows up -- each container represents a complete software
+installation that you can use and access without any extra work!
+
+The easiest way to find containers is to search on Docker Hub, but sometimes
+software pages have a link to their containers from their home page.
+
+> ## What container is right for you?
```
Comment (sstevens2) [inline _episodes/04-docker-hub.md:125]: Love this!

File: _episodes/04-docker-hub.md (hunk 1 above)
Comment (ChristinaLK) [reply _episodes/04-docker-hub.md:79]: sounds good, will tweak.

**Pull request merged**